### PR TITLE
PowerShell Error Handling Module - closes #175

### LINKED ISSUE #175: PowerShell Error Handling Module

### PR DESCRIPTION
**Pull request by jordan-grindrod**

### CODE REVIEW

File: topics/powershell/modules/error-handling/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    [CmdletBinding()]
+    param($Name)
+
+    Write-Error -Message "Nothing to write to console"
+}
+```
+
+4. Type `Hello` in the console - you should see an error message `Nothing to write to console`
+5. Type `Hello` again, but this time try to not have an error message shown
+6.Check the count of the `$error` variable again
```
Comment (bob-crutchley) [inline topics/powershell/modules/error-handling/README.md]:
```suggestion
6. Check the count of the `$error` variable again
```

File: topics/powershell/modules/error-handling/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+7. Use `$error[0]` to see what is inside the variable
```
Comment (bob-crutchley) [inline topics/powershell/modules/error-handling/README.md]: Is the error at `0` the latest error or the first one?

File: topics/powershell/modules/error-handling/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+function Hello{
+    [CmdletBinding()]
+    param($Name)
+
+    Write-Error -Message "Nothing to write to console"
+}
+```
+
+4. Type `Hello` in the console - you should see an error message `Nothing to write to console`
+5. Type `Hello` again, but this time try to not have an error message shown
```
Comment (bob-crutchley) [inline topics/powershell/modules/error-handling/README.md]: Not really sure what this means, am I supposed to give `Hello` parameter to stop the error?
Is it going to give an error regardless?

Either way I couldn't see a way to stop it giving an error unless a `Try` `Catch` is used.

File: topics/powershell/modules/error-handling/README.md (hunk 2 above)
Comment (jordan-grindrod) [reply topics/powershell/modules/error-handling/README.md]: it's the first one, which in this case should be the lastest one. Can clear up tho

File: topics/powershell/modules/error-handling/README.md (hunk 3 above)
Comment (jordan-grindrod) [reply topics/powershell/modules/error-handling/README.md]: you need to use SilentlyContinue but i will make it explicit so it;s easier

**Pull request merged**